modules/Hoot/ui/sidebar/layerConflate.js

```diff
@@ -71,6 +71,11 @@ class LayerConflate extends SidebarForm {
         this.fieldset = new FormFactory().createFieldSets( this.innerWrapper, this.formData );
     }
 
+    updateLayerColors(newColor, refType) {
+        this.fieldset.selectAll(`#${refType}.thumb`)
+            .attr('class', `thumb round _icon data light contain inline fill-${ newColor }` );
```

Comment:
This kind of approach is a bit more wordy, but will avoid duplicating the entire `class` list for the thumbnail.
```
        let thumb = this.fieldset.select(`.${refType}.thumb`);
        let oldColor = thumb.attr('class').split(' ').find(d => d.startsWith('fill-'));
        thumb.classed(`${ oldColor }`, false);
        thumb.classed(`fill-${ newColor }`, true);
```